repairs/public/js/warranty_claim.js

```diff
@@ -132,18 +132,31 @@ frappe.ui.form.on("Warranty Claim", {
 								if (["cable_status", "testing_details"].includes(result)) {
 									frm.set_value(result, data[result]);
 								} else {
-									let issue_details = result.split("_");
+									let [ear_side, service_desc, service_type] = result.split("_");
 
-									ear_side = issue_details[0]
 									ear_side = ear_side[0].toUpperCase() + ear_side.slice(1);  // Capitalize
 
-									issue_name = issue_details.slice(1).join(" ");
+									issue_name = [service_desc, service_type].join(" ");
 									issue_name = issue_name[0].toUpperCase() + issue_name.slice(1);  // Capitalize
 
+									let driver_type = service_type == "driver" ? service_desc : "";
+
+									let suggested_service;
+									frappe.db.get_value("Item Service Default", {
+										"parent": frm.doc.item_code,
+										"service_type": service_type,
+										"driver_type": driver_type
+									}, "default_service_item", (r) => {
+										if (!r.exc) {
+											suggested_service = r.default_service_item;
```

Comment:
I'm having a weird scope issue here. The variable that I've defined on line 144 is not being overriden here. I was planning to use it on line 159, but it's always being set as `null`.

Reply:
so this is an async call. Where are you using suggested_service later?
If you are expecting to have this value right after calling frappe.db.get_value(...) you'll have a bad time